### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubesphere/website/issues/2629

### CODE REVIEW

**Comment by Arhell** [comment]
/assign @rayzhou2017
add & update favicons more quality

File: static/images/favicons/site.webmanifest [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Open Source Enterprise Kubernetes Platform | KubeSphere",
```
Comment (rayzhou2017) [inline static/images/favicons/site.webmanifest:2]: Is this change good for SEO?

Comment (Arhell) [reply static/images/favicons/site.webmanifest]: corrected, thanks

**Comment by Arhell** [comment]
@rayzhou2017 all check

**Pull request merged**